src/main/sensors/gyro_init.c

```diff
@@ -557,6 +557,7 @@ bool gyroInit(void)
     case DEBUG_GYRO_SCALED:
     case DEBUG_GYRO_FILTERED:
     case DEBUG_DYN_LPF:
+    case DEBUG_D_LPF:
```

Comment:
This should not be in here - `gyro.gyroDebugMode` is used to track filters in the gyro loop, and will force the unoptimised version of `sensors/gyro_filter_impl.c` to be used, which is not wanted in this case.

Reply:
Apologies; didn't see you'd commented. Done.